ui/packages/consul-ui/vendor/consul-ui/services.js

```diff
@@ -1,21 +1,25 @@
-(services => services({
-  "route:basic": {
-    "class": "consul-ui/routing/route"
-  },
-  "service:intl": {
-    "class": "consul-ui/services/i18n"
-  },
-  "service:state": {
-    "class": "consul-ui/services/state-with-charts"
-  },
-  "auth-provider:oidc-with-url": {
-    "class": "consul-ui/services/auth-providers/oauth2-code-with-url-provider"
-  }
-}))(
-  (json, data = document.currentScript.dataset) => {
-    const appNameJS = data.appName.split('-')
-      .map((item, i) => i ? `${item.substr(0, 1).toUpperCase()}${item.substr(1)}` : item)
-      .join('');
-    data[`${appNameJS}Services`] = JSON.stringify(json);
+(services =>
+  services({
+    'route:basic': {
+      class: 'consul-ui/routing/route',
+    },
+    'service:intl': {
+      class: 'consul-ui/services/i18n',
+    },
+    'service:state': {
+      class: 'consul-ui/services/state-with-charts',
+    },
+    'auth-provider:oidc-with-url': {
+      class: 'consul-ui/services/auth-providers/oauth2-code-with-url-provider',
+    },
+    'component:consul/partition/selector': {
+      class: '@glimmer/component',
+    },
```

Comment:
By default (if partitions aren't enabled) the `Consul::Partition::Selector` is just an empty component (an direct instance of `@glimmer/component`)

There are also some other injections we need here, but all of this file is _always_ loaded i.e. these are production injections.